docs/source/API/core_objects/index.rst

```diff
@@ -1,12 +1,48 @@
-.. _coreobjects
-
 Core Objects
 ============
 
+The top-level object in a starfish workflow is the :ref:`Experiment`. It is composed of one or more
+:ref:`FieldOfView` objects, and a :ref:`Codebook`, which maps detected spots to the entities they
```

Comment:
s/spots/features/

Reply:
I don't think we actually generalize beyond spots at the moment. @dganguli ?